backend-py/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM python:3.9 as backend-py
+FROM python:3.9-slim as backend-py
```

Comment:
It takes so much shorter to pull this image that then full one making local dev a little better.